help/view-a-messages-edit-history.md

```diff
@@ -18,20 +18,10 @@ Organization administrators can
 
 1. Click on the message's **EDITED** or **MOVED** label.
```

Comment:
Since this PR removes the other subheading, let's remove `### Via the main message feed` too, and make the instruction more explicit:

```suggestion
1. Click on the message's **EDITED** or **MOVED** label shown next to the user's name in the message feed.
```

Reply:
Sure, changed it.

Reply:
@palashb01  (for future reference, it's helpful for the reviewer if you update the screenshots too)